elastio-s3-changelog/cloudformation-nested.yaml

```diff
@@ -0,0 +1,31 @@
+# This file is not intended to be used on its own. It exists to work around a bug in CloudFormation.
+# The bug prevents updating the CFN stack containing Fn::ForEach in the AWS console. This is because
```

Comment:
I wouldn't call it a bug. It's rather a "limitation" of CFN, which is documented. A bug, on the other hand, is something that contradicts the documented behavior of the system, unless it's explicitly documented as "a known bug/issue".

Reply:
I change the wording, but I disagree that this is documented. The docs only mention the `--use-previous-template` option in the `UpdateStack` call. Nothing about the Console (I understand that the Console probably uses the UpdateStack with that option, but this is still an unexpected behavior in my opinion).